tekton/build_pipeline.go

```diff
@@ -46,6 +46,10 @@ func AnnotateBuildPipelineRunWithCreateSnapshotAnnotation(ctx context.Context, p
 	message := ""
 	status := ""
 	if ensureSnapshotExistsErr == nil {
+		if !metadata.HasAnnotation(pipelineRun, SnapshotNameLabel) {
+			// do nothing fo in progress build PLR
```

Comment:
nit: `do nothing fo in progress build PLR` -> `do nothing for in progress build PLR`

Reply:
Thanks! Updated.